src/Analyzers/CSharp/Tests/UseCollectionInitializer/UseCollectionInitializerTests_CollectionExpression.cs

```diff
@@ -2123,7 +2123,7 @@ public class Goo
                 public void M()
                 {
                     int lastItem;
-                    List<int> list = [(lastItem = 5)];
+                    List<int> list = [lastItem = 5];
```

Comment:
this is an improvement.  the latter is legal in a collection expression  (but was not legal in a collection initializer).  No longer sharing code between the two improved this.